.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: node_js
```

Comment:
consider adding a link to https://docs.travis-ci.com/user/languages/javascript-with-nodejs/ for future maintainers...

Reply:
Good idea.